code/clue.js

```diff
@@ -1,59 +1,359 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+// STEP 1 - KILLER
 
 const mrGreen = {
-  firstName: 'Jacob',
-  lastName: 'Green',
-  color: 'green',
-  description: 'He has a lot of connections',
+  firstName: "Jacob",
+  lastName: "Green",
+  color: "green",
+  description: "He is an entrepeneur with an illegal hobby",
   age: 45,
-  image: 'assets/green.png',
-  occupation: 'Entrepreneur'
-}
+  image: "image/killer/green.jpg",
+  occupation: "Entrepreneur",
+  favouriteWeapon: "Pistol",
+};
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const colonelMustard = {
+  firstName: "Jack",
+  lastName: " Mustard",
+  color: "yellow",
+  description: "He is the most dangerous military man in the country",
+  age: 55,
+  image: "image/killer/mustard.jpg",
+  occupation: "Military man",
+  favouriteWeapon: "dumbell",
+};
+
+const mrsPeacock = {
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  color: "blue",
+  description: "Her husband is a famous politician",
+  age: 76,
+  image: "image/killer/peacock.jpg",
+  occupation: "Art collector",
+  favouriteWeapon: "knife",
+};
+
+const plum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  color: "purple",
+  description: "He is an professor in criminology",
+  age: 56,
+  image: "image/killer/plum.jpg",
+  occupation: "Professor",
+  favouriteWeapon: "candlestick",
+};
+const scarlet = {
+  firstName: "Cassandra",
+  lastName: "Scarlet",
+  color: "red",
+  description: "She has a rich dangerous husband ",
+  age: 43,
+  image: "image/killer/scarlet.jpg",
+  occupation: "Trophy-wife",
+  favouriteWeapon: "baseball bat",
+};
+
+const wadsworth = {
+  firstName: "Mr",
+  lastName: "Boddy",
+  color: "grey",
+  description: "Is really sneaky and trusts noone",
+  age: 49,
+  image: "image/killer/wadsworth.jpg",
+  occupation: "Stockbroker",
+  favouriteWeapon: "gun",
+};
+
+const mrsWhite = {
+  firstName: "Mrs",
+  lastName: "White",
+  color: "white",
+  description: "She knows the house like the back of her hand",
+  age: 52,
+  image: "image/killer/white.jpg",
+  occupation: "Chef",
+  favouriteWeapon: "rope",
+};
+
+const yvette = {
+  firstName: "Miss",
+  lastName: "Yvette",
+  color: "brown",
+  description: "Is probably the most evil villain in the house",
+  age: 36,
+  image: "image/killer/yvette.jpg",
+  occupation: "Maid",
+  favouriteWeapon: "poison",
```

Comment:
Since you are not using the favouriteWeapon to randomly assign one (red level) maybe this could be removed?